--- /work/vendor/stock/analysis/extracted/boot-fit-fdt.dts	2026-08-31 05:23:14.084143222 +0000
+++ /work/vendor/stock/live-device-tree.dts	2026-08-31 05:23:13.557142327 +0000
@@ -1,12 +1,19 @@
 /dts-v1/;
 
+/memreserve/	0x0000000000063000 0x000000000000b000;
 / {
+	serial-number = "e6a45d5007608a23";
 	#address-cells = <0x01>;
 	#size-cells = <0x01>;
 	compatible = "vendor,tl3506-evm-emmc-ec100\0rockchip,rk3506";
 	interrupt-parent = <0x01>;
 	model = "RK3506-EVM-EMMC-EC100";
 
+	memory {
+		reg = <0x00 0x20000000 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00>;
+		device_type = "memory";
+	};
+
 	aliases {
 		gpio0 = "/pinctrl/gpio@ff940000";
 		gpio1 = "/pinctrl/gpio@ff870000";
@@ -293,7 +300,7 @@
 
 		drm-logo@0 {
 			compatible = "rockchip,drm-logo";
-			reg = <0x00 0x00>;
+			reg = <0x1d900000 0xb8000>;
 			phandle = <0x74>;
 		};
 
@@ -1776,6 +1783,23 @@
 		route {
 
 			route-dsi {
+				overscan,bottom_margin = <0x64>;
+				overscan,top_margin = <0x64>;
+				overscan,right_margin = <0x64>;
+				overscan,left_margin = <0x64>;
+				video,aspect_ratio = <0x00>;
+				video,flags = <0x06>;
+				video,vrefresh = <0x3c>;
+				video,crtc_vsync_end = <0x2da>;
+				video,crtc_hsync_end = <0x596>;
+				video,vdisplay = <0x2d0>;
+				video,hdisplay = <0x500>;
+				video,clock = <0x1220a>;
+				logo,ymirror = <0x00>;
+				logo,bpp = <0x10>;
+				logo,height = <0x10e>;
+				logo,width = <0x28e>;
+				logo,offset = <0x61000>;
 				status = "okay";
 				logo,uboot = "logo.bmp";
 				logo,kernel = "logo_kernel.bmp";
@@ -1817,6 +1841,7 @@
 	};
 
 	ethernet@ff4c8000 {
+		local-mac-address = [ee 8a 55 bb 85 41];
 		compatible = "rockchip,rk3506-gmac\0snps,dwmac-4.20a";
 		reg = <0xff4c8000 0x2000>;
 		interrupts = <0x00 0x42 0x04 0x00 0x45 0x04>;
@@ -1880,6 +1905,7 @@
 	};
 
 	ethernet@ff4d0000 {
+		local-mac-address = [f2 8a 55 bb 85 41];
 		compatible = "rockchip,rk3506-gmac\0snps,dwmac-4.20a";
 		reg = <0xff4d0000 0x2000>;
 		interrupts = <0x00 0x46 0x04 0x00 0x49 0x04>;
@@ -2087,7 +2113,7 @@
 	};
 
 	chosen {
-		bootargs = "console=ttyFIQ0 root=PARTUUID=614e0000-0000-4b53-8000-1d28000054a9 rootwait rw";
+		bootargs = "storagemedia=emmc androidboot.storagemedia=emmc androidboot.mode=normal  rootwait console=ttyFIQ0 root=/dev/mmcblk0p6 androidboot.fwver=ddr-v1.06-d27ac532c4,tee-v2.10,uboot-03/20/2026";
 	};
 
 	acodec-sound {
